# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/interfaces/moz.build
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/interfaces/moz.build
--- a/accessible/interfaces/moz.build	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/interfaces/moz.build	Wed Nov 29 03:53:20 2023 +0000
@@ -40,7 +40,6 @@
     "nsIAccessibleTextSelectionChangeEvent.idl",
     "nsIAccessibleTypes.idl",
     "nsIAccessibleValue.idl",
-    "nsIAccessibleVirtualCursorChangeEvent.idl",
 ]
 
 XPIDL_MODULE = "accessibility"